### DISCUSSION TITLE
Ubuntu PPA outdated

### DISCUSSION THREAD

**Discussion by MarcelCoding** [abraunegg/onedrive#1478] [Q&A] [answered]
Hey @yann1cks,

I am using your ppa (https://launchpad.net/~yann1ck/+archive/ubuntu/onedrive) for OneDrive on Ubuntu. However your ppa is a patch behind. Would you like to update it?

I am using Ubuntu 20.04.1.

Best regards

**Comment by abraunegg** [comment, marked as answer]
@MarcelCoding 
You must ask this on Ubuntu Launchpad / lodge a bug there. This is not the place to ask @yann1cks to upgrade the PPA.
